### ISSUE TITLE
Question about result reproduction

### ISSUE THREAD

**Issue by zhengjin11** [2644521362/Cdcca#2]
The work presented in your paper is truly impressive. However, I encountered some difficulties while attempting to reproduce your experimental results. Would it be possible to seek your guidance on this matter?
My question is based on the latest commit: bd865a7612d1b43f17d5b171e206b478cb19ce47.

## Q1: Possible code error?
In **exps/finetune/mm/inference_13B.sh**, why is the JSON configuration file for the 7B model being loaded?

```
#!/bin/bash
PRETRAINED=/home/cx/ckpts/TPAMI/common/llama2_qformer_7B_aokvqa/epoch2
PRETRAINED_BASE=/home/cx/ckpts/llama2_acc/alpacaLlava_llamaQformerv2
LLAMA_CONFIG="/home/cx/llama2_accessory/LLaMA2-Accessory-main/accessory/configs/7B_params.json"
TOKENIZER=/home/cx/llama2_accessory/LLaMA2-Accessory-main/accessory/configs/tokenizer.model
data_parallel=fsdp
model_parallel=1
CUDA_VISIBLE_DEVICES=5 torchrun --nproc-per-node=1 --master-port=11113 /home/cx/llama2_accessory/LLaMA2-Accessory-main/accessory/demos/single_turn_mm_cloud.py \
--pretrained_path $PRETRAINED --pretrained_path_base $PRETRAINED_BASE --llama_type llama_qformerv2_cloud --llama_config $LLAMA_CONFIG --tokenizer_path $TOKENIZER
```

In **demos/single_turn_mm_cloud.py**, what is the difference between with args.pretrained_path_base and args.pretrained_path. Why do we need to load the model twice?

```
with default_tensor_type(dtype=target_dtype, device="cpu" if args.quant else "cuda"):
    model = MetaModel(args.llama_type, args.llama_config, args.tokenizer_path, with_visual=True)
#exit()
print(f"load pretrained from {args.pretrained_path}")
load_result = load_tensor_parallel_model_list(model, args.pretrained_path_base)
# print("load result: ", load_result)
load_result = load_tensor_parallel_model_list(model, args.pretrained_path)
# print("load result: ", load_result)
```

## Q2: Missing Implementation Code for Monte Carlo Dropout

In the UTS Stage 2 described in the original paper,  you employed the Monte Carlo Dropout technique to filter tokens from the uncertainty samples and upload them to the cloud. Experimental results demonstrate that Monte Carlo Dropout effectively reduces the amount of data uploaded to the cloud. But in **demos/single_turn_mm_edge.py**,  the provided code does not implement Monte Carlo Dropout—it merely filters samples based on a basic uncertainty metric.

```
#UTS-1 & UTS-2
if conf < 0.5:
    print("something!")
    # 可定义在外部或提前初始化
    sample = {
        "img_path": img_path,
        "question": question,
        "token_tensor": image_tokens,
        "prob": conf
    }
    failed_samples.append(sample)
```

## Q3: About VQAv2-to-AOKVQA experiments
The paper's Section 4.2 contains the following assertion: "Specifically, we adopt VQA-v2 [10] to finetune the pre-trained MLLM, LLaMA-Adapter (7B & 13B). Then, the VQA accuracy results on A-OKVQA [11] under different conditions (multiple choices (MC) & direct answers (DA)) are evaluated and recorded in Table 1"

How can I obtain the LLaMA-Adapter (7B & 13B) fine-tuned on VQA-v2? Could you provide a reference script? Without this, it seems difficult to reproduce the subsequent results on A-OKVQA.

Current Obstacles:
- The LLaMA-Adapter (7B & 13B) fine-tuned on VQA-v2 is unavailable in public repositories.
- Without them, the A-OKVQA reproduction pipeline is incomplete since: A-OKVQA performance builds upon VQA-v2 adaptation
